gui/mozregui/wizard.py

```diff
@@ -244,6 +256,8 @@ def options(self):
         else:
             kind = "changeset"
             fetch_config.set_inbound_branch(options['inbound_branch'])
+        if options['build_type'] is not 'opt':
```

Comment:
Hm, there is an issue there. If we only choose a nightly build, options['build_type'] will be '' and that will fail. Plus we don't need to make an exception for opt. Please just replace:
```
if options['build_type'] is not 'opt':
```
with:
```
if options['build_type']:
```